main_tab.py

```
@@ -0,0 +1,567 @@
+import sys
+import json
+from datetime import datetime as dt_class, datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)